core/modules/layout/layout.admin.inc

```diff
@@ -2653,75 +2654,86 @@ function layout_ajax_form_update($form, $form_state) {
-
-  // Get the list of layout template options. The list needs to be rebuilt (see
-  // https://github.com/backdrop/backdrop-issues/issues/984)
-  $all_template_info = layout_get_layout_template_info(NULL, TRUE);
+  $rows = array();
   $options = array();
   $default = array();
   $form_state['hidden_templates'] = array();
+  $all_template_info = layout_get_layout_template_info(NULL, TRUE);
+
   foreach ($all_template_info as $template_name => $template_info) {
     // Build the default values list.
+    $is_default = FALSE;
     if (!in_array($template_name, $excluded)) {
-      $default[] = $template_name;
+      $is_default = $default[$template_name] = TRUE;
     }
 
     // Build the list of possible options, either enabled or not hidden.
-    if (in_array($template_name, $default) || !$template_info['hidden']) {
-      $options[$template_name] = theme('layout_template_option', array('template_info' => $template_info));
+    if ($is_default || !$template_info['hidden']) {
+      $rows[$template_name] = array(
+        'layout_template_info' => theme('layout_template_info', array('template_info' => $template_info)),
+        'type' => !empty($template_info['flexible']) ? t('Flexible') : t('Standard'),
+      );
+      $rows[$template_name]['operations']['data'] = array(
+        '#type' => 'dropbutton',
+        '#links' => _layout_settings_get_group_operations($template_info, $is_default),
+       );
     }
     // For disabled and hidden templates, save the fact they are disabled.
     elseif ($template_info['hidden']) {
       $form_state['hidden_templates'][] = $template_name;
     }
   }
 
-  $form['templates'] = array(
-    '#title' => t('Available layout templates'),
-    '#description' => t('Only templates that are selected here will be available when configuring layouts.'),
-    '#type' => 'checkboxes',
+  $form['#attached']['css'][] = backdrop_get_path('module', 'layout') . '/css/layout.admin.css';
+  $form['table'] = array(
+    '#type' => 'tableselect',
+    '#header' => $header,
+    '#options' => $rows,
+    '#empty' => t('There are no layout templates.'),
     '#default_value' => $default,
-    '#options' => $options,
-    '#wrapper_attributes' => array('class' => array('clearfix', 'layout-options')),
+    '#attributes' => array('class' =>   array('layout-list')),
   );
 
   $form['actions']['#type'] = 'actions';
   $form['actions']['submit'] = array(
     '#type' => 'submit',
     '#value' => t('Save configuration'),
   );
+
   return $form;
 }
 
+
 /**
  * Validation for layout settings page.
  *
  * Ensures that active layouts are not disabled while in use.
  */
 function layout_settings_page_validate(&$form, &$form_state) {
-  $options = array_keys($form_state['values']['templates']);
-  $included = array_values($form_state['values']['templates']);
+  $options = array_keys($form_state['values']['table']);
+  $included = array_values($form_state['values']['table']);
```

Comment:
If we could leave this as `templates` that lowers risk for other modules that may have form_alter'd this form (though that's unlikely).

Reply:
Changed back.